src/Employer/Employer.Web/Controllers/Part1/TrainingController.cs

```diff
@@ -20,17 +20,28 @@ public TrainingController(TrainingOrchestrator orchestrator)
         }
         
         [HttpGet("training", Name = RouteNames.Training_Get)]
-        public async Task<IActionResult> Training(VacancyRouteModel vrm, [FromQuery] string wizard = "true")
+        public async Task<IActionResult> Training(VacancyRouteModel vrm, [FromQuery] string wizard = "true", [FromQuery] string clear = "", [FromQuery] string hasTraining = "")
         {
-            var vm = await _orchestrator.GetTrainingViewModelAsync(vrm);
+            var vm = await _orchestrator.GetTrainingViewModelAsync(vrm, User.ToVacancyUser());
+
+            if (vm.IsUsersFirstVacancy && string.IsNullOrEmpty(hasTraining))
```

Comment:
First time clicking clear - This is fixed in ER-1337
Clear being highlighted but requiring extra click - it seems to do this will all element including the button. I think this is a deficiency of the accessible auto suggest library. I'll bring ti up with Richard.
Training  provider cancel link - yep Hema picked this one up. Its fixed now
Content - Will do a final sweep of content once tickets have been tested. Hema is keeping track of discrepancies.